dashboard/client/src/pages/FunctionLogPage.jsx

```diff
@@ -25,18 +25,23 @@ export class FunctionLogPage extends Component {
       log: '',
       functionName,
       user,
-      fetchError: false
+      fetchError: false,
+      numLines: 0,
```

Comment:
This should be a sensible default, say 100?

Reply:
Will make that change